kubetest/main.go

```diff
@@ -42,7 +42,7 @@ const defaultGinkgoParallel = 25
 var (
 	artifacts    = initPath("./_artifacts")
 	interrupt    = time.NewTimer(time.Duration(0)) // interrupt testing at this time.
-	kubetestPath = initPath(os.Args[0])
+	kubetestPath = os.Args[0]
```

Comment:
this is probably need to be `$WORKSPACE/kubetest` instead.

Reply:
```
root@9aafb1c3-a254-11e7-860f-0a580a6c0208:/workspace# ls   
_artifacts     e2e-runner.sh	get-kube.sh		 k8s.io		     kubernetes		  kubetest
build-log.txt  gcloud-info.txt	google-cloud-sdk.tar.gz  kops-e2e-runner.sh  kubernetes-anywhere  test-infra
```

and https://github.com/kubernetes/test-infra/blob/master/jenkins/e2e-image/Dockerfile#L64-L68